template/pdk.gen.go.ejs

```diff
@@ -14,26 +14,24 @@ func _<%- goName(ex.name) %>() int32 {
 	var err error
 	_ = err
   <% if (ex.input) { -%>
-    <% if (ex.input.contentType === 'application/json') { -%>
+    <% if (isJsonEncoded(ex.input)) { -%>
 			pdk.Log(pdk.LogDebug, "<%- goName(ex.name) %>: getting JSON input")
 			var input <%- toGolangType(ex.input) %>
 			err = pdk.InputJSON(&input)
 			if err != nil {
 				pdk.SetError(err)
 				return -1
 			}
-    <% } else if (ex.input.type === 'string') { -%>
-      <% if (ex.input.$ref && ex.input.$ref.enum) { -%>
```

Comment:
I updated all these to use the helpers so we're not exposing the properties we are going to remove. example `type` and `$ref`, etc